packages/strapi/lib/core/configurations.js

```diff
@@ -270,13 +270,24 @@ module.exports.app = async function() {
 
 const enableHookNestedDependencies = function (name, flattenHooksConfig) {
   if (!this.hook[name]) {
-      this.log.warn(`(hook:${name}) \`strapi-${name}\` is missing in your dependencies. Please run \`npm install strapi-${name}\``);
+    this.log.warn(`(hook:${name}) \`strapi-${name}\` is missing in your dependencies. Please run \`npm install strapi-${name}\``);
   }
 
   // Couldn't find configurations for this hook.
   if (isEmpty(get(flattenHooksConfig, name, true))) {
+
+    // Check if database connector is used
+    let isUsed = false;
+    forEach(get(strapi, 'api', {}), api => {
```

Comment:
You don't need to use `get` lodash method, it will be better to use `this` instead of the global variable and I'm still not a big fan of `let` variable update in a forEach scope (but it's just my opinion 🙈)

```js
const modelUsed = Object.keys(this.api || {})
  .filter(x => isObject(this.api[x].models)) // Filter API with models
  .map(x => x.models) // Keep models
  .filter(model => get(this.connection, model.connection, {}).connector === name)); // Filter model with the right connector

flattenHooksConfig[name] = {
  enabled: modelUsed.length > 0 // Will return false if there is no model, else true.
};
```